python/nemo_relay/llm.py

```
@@ -382,7 +387,7 @@ def finalize():
     )
 
 
-def request_intercepts(name, request):
+def request_intercepts(name: str, request: LLMRequest) -> Any:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `request_intercepts`

(ANN401)